src/app/clusters/tls-client-management-server/BUILD.gn

```diff
@@ -14,7 +14,11 @@
 import("//build_overrides/chip.gni")
 
 source_set("tls-client-management-server") {
-  sources = [ "TlsClientManagementCluster.h" ]
+  sources = [
+    "CodegenIntegration.cpp",
```

Comment:
CodegenIntegration is ember specific. it should not be here and must be in the app specific files only